VMBackup/main/freezesnapshotter.py

```diff
@@ -178,6 +177,20 @@ def takeSnapshotFromGuest(self):
                             self.hutil.SetExtErrorCode(ExtensionErrorCodeHelper.ExtensionErrorCodeEnum.FailedRetryableSnapshotFailedRestrictedNetwork)
                             error_msg = error_msg + ExtensionErrorCodeHelper.ExtensionErrorCodeHelper.StatusCodeStringBuilder(self.hutil.ExtErrorCode)
                         run_status = 'error'
+
+                        #making retryble error incase of intermittent internal failure in snapshot
+                        #making FailedSnapshotLimitReached error incase of "snapshot blob calls is exceeded"
+                        for snapshot_info in snapshot_info_array:
+                            if "The rate of snapshot blob calls is exceeded" in snapshot_info.errorMessage :
```

Comment:
add or for "The snapshot count against this blob has been exceeded" as well as this is also sometimes received, i.e.
or "The snapshot count against this blob has been exceeded" in snapshot_info.errorMessage